docs/asciidoc/problem-details.adoc

```diff
@@ -10,52 +10,44 @@ Thankfully, there’s a standard called https://www.rfc-editor.org/rfc/rfc7807[I
 By adopting `RFC 7807`, API designers don’t have to spend time creating a custom solution, and users benefit by recognizing a familiar format across different APIs.
 If it suits the API’s needs, using this standard benefits both designers and users alike.
 
-`Jooby` provides built-in support for `Problem Details`. There are two main entities to work with:
+`Jooby` provides built-in support for `Problem Details`.
 
-1. `HttpProblem` - representation of the `RFC 7807` model and the way to instantiate the problem.
+=== Set up ProblemDetails
 
-2. `ProblemDetailsHandler` - global error handler that catches all exceptions, transforms them into `Problem Details` compliant format and renders the response based on the `Accept` header value.
-It also sets the appropriate content-type in response (e.g. application/problem+json, application/problem+xml)
+To enable the `ProblemDetails`, simply add the following line to your configuration:
 
-=== Set up handler
+.application.conf
+[source, properties]
+----
+problem.details.enabled = true
+----
 
-The bare minimal requirement is to set up an error handler:
+This is the bare minimal configuration you need.
+It enables a global error handler that catches all exceptions, transforms them into Problem Details compliant format and renders the response based on the Accept header value. It also sets the appropriate content-type in response (e.g. application/problem+json, application/problem+xml)
 
-.Java
-[source,java,role="primary"]
-----
-import io.jooby.problem.ProblemDetailsHandler;
+All supported settings include:
 
-{
-  ...  
-  error(new ProblemDetailsHandler()          // <1>
-    .log4xxErrors()                          // <2>
-    .mute(StatusCode.UNAUTHORIZED)           // <3>
-  );
+.application.conf
+[source, properties]
+----
+problem.details {
+  enabled = true
+  log4xxErrors = true                                // <1>
+  muteCodes = [401, 403]                             // <2>
+  muteTypes = ["com.example.MyMutedException"]       // <3>
 }
 ----
 
-.Kotlin
-[source,kt,role="secondary"]
-----
-import io.jooby.problem.ProblemDetailsHandler
 
-{
-  ...  
-  error(new ProblemDetailsHandler()          // <1>
-    .log4xxErrors()                          // <2>
-    .mute(StatusCode.UNAUTHORIZED)           // <3>
-  )
-}
-----
+<1> By default, only server errors (5xx) will be logged. You can optionally enable the logging of client errors (4xx). If `DEBUG` logging level is enabled, the log will contain a stacktrace as well.
+<2> You can optionally mute some status codes completely.
```

Comment:
Let's add a note about what `mute` means, like `debug` log statement.

Reply:
The description is actually correct. `ProblemDetailsHandler` redefines the logging approach. Muted codes/types aren't logged under any condition, and here is why. From the app perspective the reason for `mute` can be some sort of security requirement. Let's take a look at the RFC7807 example   
![image](https://github.com/user-attachments/assets/802715fb-cbbb-4a87-b879-6b7ec84e8be8)

I'm not sure if such sensitive user data like an account and its balance should be logged even under the `debug` level. That's why `mute` here means mute completely.

But we still have enough options to control logging:
- `log4xxErrors` flag to enable the logging of all client errors
- with `DEBUG` level the stack trace is also included when  some investigation is needed
- `mute` as a pinpoint mechanism to hide sensitive or annoying logs completely

So, do you want me to return the `debug` logging for `mute`?